BlueprintUI/Tests/AccessibililtyTraitsTests.swift

```diff
@@ -0,0 +1,35 @@
+import BlueprintUI
+import UIKit
+import XCTest
+
+
+class UIAccessibilityTraits_Tests: XCTestCase {
+
+    func test_backButton() throws {
+
+        let root = UIViewController()
+        let second = UIViewController()
+        let navigationController = UINavigationController()
+        navigationController.viewControllers = [root, second]
+
+        try show(vc: navigationController) { _ in
+            let button = try XCTUnwrap(
+                navigationController.navigationBar.findSubview(ofType: UIControl.self)
+            )
+
+            /// The `.backButton` trait is private, so this test is here to verify it does not change.
+
+            XCTAssertEqual(button.accessibilityTraits, .backButton)
+        }
+    }
+
+    func test_toggleSwitch() {
+        if #available(iOS 17.0, *) {
+            /// The `.toggleButton` trait is exposed iOS 17, so this test is here to verify it matches ours.
+            XCTAssertEqual(UIAccessibilityTraits._toggleButton, UIAccessibilityTraits.toggleButton)
+        }
+        let uiSwitch = UISwitch()
+        /// The `.toggleButton` trait is private prior to iOS 17, so this test is here to verify it exists on prior versions.
+        XCTAssertTrue(uiSwitch.accessibilityTraits.contains(._toggleButton))
```

Comment:
Its wild to me that this test in particular is failing. I suspect its an issue with the blueprint test configuration. 
This passes in isolation with a freshly generated project.

Reply:
the test failures show the UISwitch having  UIAccessibilityTraits.None which is obviously wrong.

Reply:
Do the tests have an app host? I imagine that may be required